### PR TITLE
Show template average build

### PR DESCRIPTION
**Pull request by ammario**
Continuing @iskhakovt's work from https://github.com/coder/coder/pull/3999

### CODE REVIEW

**Comment by ammario** [comment]
I have read the CLA Document and I hereby sign the CLA

File: site/src/pages/TemplatesPage/TemplatesPageView.tsx [hunk 1]
```diff
@@ -38,12 +39,17 @@ import {
...
+    buildTimeSec === -1
+      ? "Unknown"
+      : dayjs.duration(buildTimeSec, "seconds").humanize(),
   developerCount: (activeCount: number): string => {
     return `${formatTemplateActiveDevelopers(activeCount)} developer${
       activeCount !== 1 ? "s" : ""
     }`
   },
   nameLabel: "Name",
+  buildTimeLabel: "Build time",
```
Comment (Kira-Pilot) [inline site/src/pages/TemplatesPage/TemplatesPageView.tsx:52]: We'll want to move this in to a translation file - `site/src/i18n/en/templatesPage.json` will work fine.
You can look at[ this explanatory PR](https://github.com/coder/coder/pull/3682) to see how how use translations in the component file.

File: site/src/pages/TemplatesPage/TemplatesPageView.tsx [hunk 2]
```diff
@@ -38,12 +39,17 @@ import {
 } from "../../components/Tooltips/HelpTooltip/HelpTooltip"
 
 export const Language = {
+  buildTime: (buildTimeSec: number): string =>
+    buildTimeSec === -1
+      ? "Unknown"
+      : dayjs.duration(buildTimeSec, "seconds").humanize(),
```
Comment (Kira-Pilot) [inline site/src/pages/TemplatesPage/TemplatesPageView.tsx:45]: We'll want to translate "Unknown" here

File: site/src/pages/TemplatesPage/TemplatesPageView.tsx (hunk 1 above)
Comment (Kira-Pilot) [reply site/src/pages/TemplatesPage/TemplatesPageView.tsx:52]: Where are these strings actually being used?

**Comment by ammario** [comment]
Continuing this here https://github.com/coder/coder/pull/4557

File: site/src/pages/TemplatesPage/TemplatesPageView.tsx (hunk 1 above)
Comment (ammario) [reply site/src/pages/TemplatesPage/TemplatesPageView.tsx:52]: I think if we want to use translations across the board we should migrate everything in one fell swoop.

Comment (Kira-Pilot) [reply site/src/pages/TemplatesPage/TemplatesPageView.tsx:52]: I'm happy to handle that; maybe toward the end of next week.

Comment (ammario) [reply site/src/pages/TemplatesPage/TemplatesPageView.tsx:52]: Up to you. I have no preference, except for consistency.

**Pull request closed without merging**